components/touchscreen/cst226.rst

```diff
@@ -27,18 +27,33 @@ Base Touchscreen Configuration
     touchscreen:
       platform: cst226
       id: my_touchscreen
-      interrupt_pin: GPIO3
+      interrupt_pin: REPLACEME
 
 Configuration variables:
 ************************
 
 - **id** (*Optional*, :ref:`config-id`): Manually set the ID of this touchscreen.
-- **interrupt_pin** (**Required**, :ref:`Pin Schema <config-pin_schema>`): The touch detection pin.
-- **rotation** (*Optional*): Set the rotation of the touchscreen. By default this will be set to match the display associated with the touchscreen, but this allows more control. Choices are ``0``, ``90``, ``180`` and ``270``.
+- **interrupt_pin** (*Optional*, :ref:`Pin Schema <config-pin_schema>`): The touch detection pin.
+- **reset_pin** (*Optional*, :ref:`Pin Schema <config-pin_schema>`): The chip reset pin.
 
 - All other options from :ref:`Touchscreen <config-touchscreen>`.
 
 
+Sample config for the T4-S3
+***************************
+
+.. code-block:: yaml
+
+    i2c:
+      sda: 6
```

Comment:
```suggestion
      sda: GPIO6
```

BTW. should this not be **REPLACEME** as well? Or has this to do that this is a T4-S3 example.

Reply:
This is a specific example for the T4-S3. The generic minimal config above has REPLACEME.